### PR TITLE
DOC Deprecate TopPage classes that are being renamed

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/issues/1240

### CODE REVIEW

File: en/08_Changelogs/5.4.0.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- [Features and enhancements](#features-and-enhancements)
+  - [Other new features](#other-new-features)
+- [API changes](#api-changes)
+- [Bug fixes](#bug-fixes)
+
+## Features and enhancements
+
+### Other new features
+
+- (fill this is as features are added)
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.4.0.md:18]: Typo here but it's a placeholder so doesn't really matter.

File: en/08_Changelogs/5.4.0.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## API changes
+
+- The class names for the `TopPage` feature in [`dnadesign/silverstripe-elemental`](https://github.com/silverstripe/silverstripe-elemental) do not follow the correct naming convention for Silverstripe CMS. The existing classes have been deprecated with will be renamed to match the correct naming convention in a future major release.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.4.0.md]:
```suggestion
- The class names for the `TopPage` feature in [`dnadesign/silverstripe-elemental`](https://github.com/silverstripe/silverstripe-elemental) do not follow the correct naming convention for Silverstripe CMS. The existing classes have been deprecated and will be renamed to match the correct naming convention in a future major release.
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Minor change

File: en/08_Changelogs/5.4.0.md (hunk 2 above)
Comment (emteknetnz) [reply en/08_Changelogs/5.4.0.md]: Done

**Review by GuySartorelli — APPROVED** [review]
Will merge once we've got 5.3.0-beta1 out of the way of the 5 branch.

**Pull request merged**